yarn-project/end-to-end/src/e2e_inclusion_proofs_contract.test.ts

```diff
@@ -29,7 +29,7 @@ describe('e2e_inclusion_proofs_contract', () => {
 
   afterAll(() => teardown());
 
-  it('creates a note and proves its existence', async () => {
+  it('proves note existence and its nullifier non-existence and nullifier non-existence failure case', async () => {
```

Comment:
how is this failure case?

Reply:
[In the test](https://github.com/AztecProtocol/aztec-packages/blob/b8a1e6c168214ab3849616e42ddd95a6c1c983eb/yarn-project/end-to-end/src/e2e_inclusion_proofs_contract.test.ts#L64) I nullify the note and I try to prove the nullifier non-inclusion again.